app/models/document_factory_registry.rb

```diff
@@ -6,7 +6,7 @@
 require "builders/manual_document_builder"
 require "manual_with_documents"
 require "slug_generator"
-require "specialist_document"
+require "section"
```

Comment:
Is this require statement still necessary now that the classname matches the filename?